test/cypress/e2e/events.cy.ts

```diff
@@ -19,6 +19,7 @@ describe('EventsPage', () => {
           {
             id: 1,
             name: 'Event One',
+            slug: 'event-one',
```

Comment:
If `slug` is an important field, make sure to add and any others you are using to `https://github.com/OpenMeet-Team/tenant-service/blob/main/tenants.yaml`. I'll need to remember to go update the prod and dev deployments with the new config.

Reply:
`slug` is used in Event, Group, User models only. It doesn't interact with tenant config